### ISSUE TITLE
Warning related to browser properties

### ISSUE THREAD

**Issue by linusfj** [hajkmap/Hajk#1349]
**Describe the bug**
The application is generating a warning in the browser related to the usage of `navigator.userAgent`, `navigator.appVersion`, and `navigator.platform`. These properties have potential privacy implications and are being phased out by modern browsers, starting with Chrome 101.

**To Reproduce**
1. Launch the application in the browser.
2. Open Developer Tools and navigate to 'Issues'.
3. Notice the warning message under the 'Other' heading.

**Expected behavior**
Ideally, the application should run without triggering any warnings, aligning with modern browser standards and privacy-focused practices.

**Additional context**
One solution could be migrating to `navigator.userAgentData`, a new method introduced in Chrome 101. Further details can be found in this [Chrome Developers Blog post](https://developer.chrome.com/blog/deps-rems-101/).

**Comment by jacobwod** [comment]
I guess the guilty portion of our code comes from [`detect-browser`](https://github.com/DamonOehlman/detect-browser) is doing. There's (at least) [one open issue about it here](https://github.com/DamonOehlman/detect-browser/issues/174), though no PR yet.

We could wait and see or, alternatively, find another solution (which shouldn't be impossible).

**Comment by jacobwod** [comment]
On a second thought: I think we should consider removing this functionality entirely. 

The reason it exists is to allow **showing [Announcements](https://github.com/hajkmap/Hajk/wiki/Experimental-features#announcements) in specific browsers only**. The only use of this I've ever seen was to trigger an Announcement for users running the now unsupported Internet Explorer 11. 

**Can we still see any reasons why Announcements should have the option to trigger on specific browsers only?**

Also ping @Hallbergs @jesade-vbg

**Comment by Hallbergs** [comment]
I think we should remove it entirely. Cannot see any reason to trigger on specific browsers anymore. (We only used it to trigger on IE).

**Comment by jacobwod** [comment]
Do you see it here? https://karta.halmstad.se/#m=ext&x=110600&y=6283796&z=2&l=58%2C100&p=layerswitcher

**Comment by linusfj** [comment]
Yes, I have tested using the provided URL, and I still get the warning message.

However, this seems to be a more general issue as I encounter the same warning message on several other sites as well.

**Comment by linusfj** [comment]
It seems that the issue has been resolved. The error no longer appears and is likely due to an update to Chrome and Edge.
